posts/hybrid-reference-profiles/index.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Creating hybrid reference profiles for InSituType"
+author:
+  - name: Patrick Danaher
+    orcid: 0000-0002-2844-5883
+    affiliations:
+      - ref: nstg
+      - ref: patrickjdanaher
+date: "2024-08-21"
+date-modified: "2024-08-21"
+categories: [cell typing]
+draft: false
+image: figures/heatmap_tile.png
+---
+
+
+## Background
+
+The InSituType cell typing algorithm relies on a "reference matrix" to perform supervised or semi-supervised cell typing.
```

Comment:
Suggest adding a link to the InSituType github and / or manuscript. Use `{target="_blank"}` so that it opens in a new tab.